packaging/docker/run.sh

```diff
@@ -67,4 +67,19 @@ if [ -n "${NETDATA_CLAIM_URL}" ] && [ -n "${NETDATA_CLAIM_TOKEN}" ] && [ ! -f /v
                              -daemon-not-running
 fi
 
+if [ -n "${NETDATA_EXTRA_APK_PACKAGES}" ]; then
+  echo "Fetching APK repository metadata."
+  if ! apk update; then
+    NETDATA_FAILED_APK_UPDATE=1
+    echo "Failed to fetch APK repository metadata."
+  fi
+fi
+
+if [ -n "${NETDATA_EXTRA_APK_PACKAES}" ] && [ -z "${NETDATA_FAILED_APK_UPDATE}" ]; then
+  echo "Installing supplementary packages."
+  if ! apk add --no-cache --no-interactive "${NETDATA_EXTRA_APK_PACKAGES}"; then
```

Comment:
```suggestion
  if ! apk add --no-cache ${NETDATA_EXTRA_APK_PACKAGES}; then
```

- `--no-interactive` doesn't exist (there is `--interactive` param).
- I think we need word splitting for `NETDATA_EXTRA_APK_PACKAGES`, otherwise
> ERROR: 'apcupsd libvirt-daemon lm-sensors msmtp netcat-openbsd' is not a valid world dependency, format is name(@tag)([<>~=]version